requirements.in

```diff
@@ -2,7 +2,7 @@ coverage
 django
 django[argon2]
 django-cors-headers
-django-enumfield==1.3b2
+django-enumfields==0.8.2
```

Comment:
You don't need to fix the version in the requirements.in file, except if you want a prerelease version or deliberately keep an old version.